HuboJointState[] joints
HuboIMU imu
HuboIMU left_foot
HuboIMU right_foot
HuboFT left_wrist
HuboFT right_wrist
HuboFT left_ankle
HuboFT right_ankle
